src/components/back-link/_back-link.scss

```diff
@@ -4,6 +4,8 @@
 
 @include govuk-exports("back-link") {
 
+  $backlink-border-colour: $govuk-black !default;
```

Comment:
Should these be namespaced with `govuk`?

Reply:
If we're expecting them to be referenced from outside of the component (e.g. by users customising the colours of a button before importing Frontend) then I think they probably should.

Is it worth holding off on doing this work until we've established our public API?

Reply:
If these are not part of the public api we should consider not allowing them to be overridden?

Reply:
i'll stop then